aspnetcore/tutorials/web-api-help-pages-using-swagger/samples/2.0/TodoApi.Swashbuckle/Startup2.cs

```diff
@@ -22,9 +22,9 @@ public void ConfigureServices(IServiceCollection services)
             services.AddMvc();
 
             // Register the Swagger generator, defining 1 or more Swagger documents
-            services.AddSwaggerGen(c =>
+            services.AddSwaggerGen(swaggerGeneratorOptions =>
```

Comment:
Let's shorten the variable name to `options`. Shorter variable names are desired to prevent horizontal scrolling.